packages/fxa-auth-server/lib/error.js

```diff
@@ -448,6 +448,20 @@ AppError.invalidNonce = function() {
   });
 };
 
+AppError.unauthorized = function unauthorized(reason) {
```

Comment:
I added it because I felt that none of the existing functions for formatting errors fully captured what needed to be returned in the event of an authorization error. It made the most sense to use the `AppError.unauthorized()` that exists in `lib/oauth/error.js` and is what I was using initially.
I duplicated it here because of the following: 
When the `AppError.translate()` was being called, the error wasn't formatting correctly because it was using the oauth version, but the endpoint being called wasn't configured as an oauth endpoint. I thought it made the most sense to create an error that accurately reflected the error encountered.